sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -671,6 +672,11 @@ export enum DataType {
     String = "String"
 }
 
+// @public
+export interface DedicatedGatewayRequestOptions {
```

Comment:
Since this is a Beta API feature, I am wondering where would we mark the `experimental` feature? on this interface or the public APIs?